example/shared/src/main/scala/scalaz/example/TheseUsage.scala

```diff
@@ -0,0 +1,65 @@
+package scalaz
```

Comment:
Could you refactor this to similar to #1537 ?

Reply:
Sure thing! I think the `tut` plugin wasn't in yet when I did this. Thanks for the reminder.

Reply:
No worries, things overlapped a little, I had the same thing with the usage of Disjunction